### PR TITLE
Small Documentation improvements 

### PR DESCRIPTION
**Pull request by Jowan-Spooner**

### CODE REVIEW

File: addons/dialogic/Core/DialogicResourceUtil.gd [hunk 1]
```diff
@@ -102,6 +106,7 @@ static func remove_resource(file_path:String) -> void:
 		key = directory.find_key(file_path)
 	set_directory(file_path.get_extension(), directory)
 
+
 static func is_identifier_unused(extension:String, identifier:String) -> bool:
```
Comment (CakeVR) [inline addons/dialogic/Core/DialogicResourceUtil.gd:110]: Missing docs on public item.

File: addons/dialogic/Core/DialogicResourceUtil.gd [hunk 2]
```diff
@@ -137,6 +142,7 @@ static func update_label_cache() -> void:
 #region EVENT CACHE
 ################################################################################
 
+## Dialogic keeps a list that has each event once. This allows retrieving that list.
```
Comment (CakeVR) [inline addons/dialogic/Core/DialogicResourceUtil.gd:145]:
```suggestion
##  Dialogic keeps a list that has each event once. This allows retrieval of that list.
```

**Review by CakeVR — CHANGES_REQUESTED** [review]
I provides some suggestions and a few comments on code passages.
There was one undocumented public GDScript item I found.

You can accept the suggestions, if you agree with them. If you have questions, please feel free to ask me.

File: addons/dialogic/Core/DialogicGameHandler.gd [hunk 3]
```diff
@@ -58,19 +60,22 @@ var paused := false:
...
 
+## Emitted when dialog ends (by a timeline end being reached OR end_timeline() being called).
 signal timeline_ended
+## Emitted when a timeline was started with [method start] or [method start_timeline].
 signal timeline_started
+## Emitted when an event has just been executed (not necessarily when the event finished).
 signal event_handled(resource:DialogicEvent)
 
-## Emitted when the Signal event was reached
+## Emitted when the `Signal Event` was reached
```
[the remark this answers is not in the document]
Comment (Jowan-Spooner) [reply addons/dialogic/Core/DialogicGameHandler.gd:76]: Well not a very good one but yeah, one is an EVENT and the other is a text effect. We can unify them though, idc

**Review by CakeVR — APPROVED** [review]

**Pull request merged**